src/main/java/models/Game.java

```diff
@@ -77,16 +77,33 @@ public void dealCardToDealer()
 		}
 	}
 
-	public void whoWon()
-	{
-		int dealerScore = dealer.calcScore(dealer.hand);
-		int userScore = user.calcScore(user.hand);
-		if (dealerScore > userScore)
-			dealerWon = true;
-		else if (dealerScore != userScore)
-			userWon = true;
+	public void whoWon(int ante)
+    {
+        int dealerScore = dealer.calcScore(dealer.hand);
+        int userScore = user.calcScore(user.hand);
+        if (dealerScore > userScore){
+            dealerWon = true;
+            user.loseBet(ante);
+        }else if (dealerScore != userScore){
```

Comment:
Do we have the new game function to "deallocate" the array and remove the users hands? But I guess that would be the empty array function, but does it actually connect?